package.json

```diff
@@ -35,11 +38,13 @@
     "react-dom": "^16",
     "react-hot-loader": "^4",
     "react-router-dom": "^4.3.1",
+    "taskcluster-client-web": "^8.1.0",
+    "taskcluster-lib-urls": "^12.0.0",
     "typeface-roboto": "^0.0.54"
   },
   "devDependencies": {
     "@neutrinojs/airbnb": "^9.0.0-beta.1",
-    "@neutrinojs/copy": "^8.3.0",
+    "@neutrinojs/copy": "^9.0.0-beta.1",
     "@neutrinojs/jest": "^9.0.0-beta.1",
     "@neutrinojs/react": "^9.0.0-beta.1",
```

Comment:
For unstable releases, I would usually pin the version to make it more explicit. In other news, I think the stable version of neutrino i out.

Reply:
I started a PR for upgrading. There are some warnings I will deal with over there.